src/web/mjs/connectors/MangaLivre.mjs

```diff
@@ -0,0 +1,113 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class MangaLivre extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'mangalivre';
+        super.label = 'Manga Livre';
+        this.tags = [ 'manga', 'portuguese', 'webtoon' ];
+        this.url = 'https://mangalivre.com';
+    }
+
+    async _getMangas() {
+        let page = 1;
+        let morePages = true;
+
+        let request, data;
```

Comment:
No need for ahead declaration, there is no performance/memory gain and in case of refactoring it needs to be changed in two places instead of one.